apps/src/gamelab/AnimationPicker/AnimationPicker.jsx

```diff
@@ -1,4 +1,4 @@
-var actions = require('./animationPickerModule');
+import * as actions from './animationPickerModule';
 var AnimationPickerBody = require('./AnimationPickerBody.jsx');
```

Comment:
I know in my personal code this is a rule that I violate sometimes, so I'm okay with making this one of our exceptions.